tests/acceptance/docker/src/ocis-base.yml

```diff
@@ -43,6 +43,11 @@ services:
       # postprocessing step
       POSTPROCESSING_STEPS: $POSTPROCESSING_STEPS
 
+      # tika
+      SEARCH_EXTRACTOR_TYPE: "tika"
+      SEARCH_EXTRACTOR_TIKA_TIKA_URL: "http://tika:9998"
+      SEARCH_EXTRACTOR_CS3SOURCE_INSECURE: "true"
+
```

Comment:
I think, this will cause ocis to exit if tika is not available (could be a case while trying to run other tests without tika).
Please, check

Reply:
@saw-jan @SwikritiT can you check it once locally. Yes the the tika envs would make ocis stop. But changes have been made to run tika when `START_TIKA` is true.